src/MEDS_DEV/models/cehrbert/finetune_cehrbert.py

```diff
@@ -0,0 +1,93 @@
+import logging
+import subprocess
+from pathlib import Path
+
+import hydra
+from omegaconf import DictConfig, OmegaConf
+
+logger = logging.getLogger(__name__)
+
+CONFIG = Path(__file__).parent / "_config.yaml"
+finetune_yaml_template = Path(__file__).parent / "cehrbert_finetune_template.yaml"
```

Comment:
This style of file loading can be brittle at times; you could consider using a [structured config](https://hydra.cc/docs/tutorials/structured_config/intro/) dataclass object instead if it poses problems in the future. No change needed now, just wanted to highlight that.